### PR TITLE
 added one question regarding round function in sql-quiz.ts.

### PR DESCRIPTION
**Pull request by 836hardik-agrawal**
Checklist:

-[ ] I have read  https://www.w3schools.com/mysql/func_mysql_round.asp -[ ]  My pull request has a descriptive title (not a vague title like Update index.md)

Checklist:

- [x ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x ] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/sql-quiz.ts [hunk 1]
```diff
@@ -554,6 +554,18 @@ const sqlQuiz = [
       "Database data can either be linked by one-to-one, one-to-many, or many-to-many relationships.",
     Link: "https://www.freecodecamp.org/news/relational-vs-nonrelational-databases-difference-between-sql-db-and-nosql-db/",
   },
+  {
+    Question:
+      "Which function rounds a number to a specified number of decimal places.?",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:559]:
```suggestion
      "In SQL, which function rounds a number to a specified number of decimal places?",
```

File: src/data/sql-quiz.ts [hunk 2]
```diff
@@ -554,6 +554,18 @@ const sqlQuiz = [
...
+    Answer: "ROUND()",
+    Distractor1: "TRUNCATE()",
+    Distractor2: "CEIL()",
+    Distractor3: "FLOOR()",
+    Explanation:
+      "The ROUND() function rounds a number to a specified number of decimal places.",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:565]:
```suggestion
      "The ROUND() function takes in two parameters(number and decimals) and rounds the number to the number of decimals specified in the 2nd parameter.",
```

File: src/data/sql-quiz.ts [hunk 3]
```diff
@@ -554,6 +554,18 @@ const sqlQuiz = [
...
+    Link: "https://www.w3schools.com/mysql/func_mysql_round.asp",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:566]: Please only use freeCodeCamp resources or official documentation.

```suggestion
    Link: "https://dev.mysql.com/doc/refman/5.7/en/precision-math-rounding.html",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by 836hardik-agrawal — COMMENTED** [review]
I have done the required changes.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**